app/views/categories/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @category.title %></h1>
+
+<ul>
+  <% @category.jobs.each do |job| %>
+    <li><%= link_to job.title, company_path(job.company) %></li>
+  <% end %>
+<%= link_to "Create New", new_category_url %>
```

Comment:
what's the difference between `new_category_url` and `net_category_path`? why use one over the other? 

i'd suggest sticking with `_path` while working within your application 👍